packages/cadl-ranch-specs/http/clients/authentication/main.cadl

```diff
@@ -0,0 +1,26 @@
+import "@cadl-lang/rest";
```

Comment:
Because authentication has to be at the service level(for now at least) I would structure the files more or less like that:
- [clients/]authentication/
  - oauth2/
  - apiKey/
  - basic/
  - multiAuth/
  - etc...